vetiver/vetiver_model.py

```diff
@@ -29,19 +29,19 @@ class VetiverModel:
         Model name or ID
     prototype_data : pd.DataFrame, np.array
         Sample of data model should expect when it is being served
-    ptype_data : pd.DataFrame, np.array
-        Deprecated, in favor of prototype_data
     versioned :
         Should the model be versioned when created?
     description : str
         A detailed description of the model.
         If omitted, a brief description will be generated.
     metadata : dict
         Other details to be saved and accessed for serving
+    **kwargs: dict
+        Deprecated parameters.
 
     Attributes
     ----------
-    ptype : pydantic.main.BaseModel
+    prototype : pydantic.main.BaseModel
```

Comment:
I don't think we need any more references to `BaseModel` (even pydantic) outside of types.py. We have `Prototype`.

Reply:
Agreed. I believe the one place we do need to use pydantic.BaseModel is at prototype creation for the single dispatch to correctly dispatch to that class, but everywhere else can be removed!